plugin-build/plugin/src/main/kotlin/com/mikepenz/aboutlibraries/plugin/BaseAboutLibrariesTask.kt

```diff
@@ -91,12 +91,10 @@ abstract class BaseAboutLibrariesTask : DefaultTask() {
     internal abstract val variantToDependencyData: MapProperty<String, List<DependencyData>>
 
     open fun configure() {
-        if (!excludeFields.isPresent) {
-            excludeFields.set(project.provider {
-                val config = extension.exports.findByName(variant.getOrElse(""))
-                config?.excludeFields?.orNull ?: extension.export.excludeFields.get()
-            })
-        }
+        excludeFields.set(project.provider {
+            val config = extension.exports.findByName(variant.getOrElse(""))
+            config?.excludeFields?.orNull?.takeIf { it.isNotEmpty() } ?: extension.export.excludeFields.get()
+        })
 
         if (!includeMetaData.isPresent) {
```

Comment:
No need to adjust this which follows the same pattern?

Reply:
I checked with logs. And get() would fall for them if not set, e.g. is present bring false. Different from the set property 🥲.